### ISSUE TITLE
Scrubs not spawning, scrubs not despawning

### ISSUE THREAD

**Issue by r0bd0g** [TestRunnerSRL/OoT-Randomizer#114]
From: Scott (hatwearinggamer)

"So I entered the Fenced in grotto in Hyrule Field near Lake Hylia for the first time and found no Deku scrub. I believe this is the first time I've been there as I do not have any form of bombs and only just found the Megaton Hammer."
Seed info: OoT_RN2D69BXUU37TBA_IBHHY977Q9
Test Runners 2.17.1

**Comment by r0bd0g** [comment]
Same seed as above.
The right-side scrub in the lost woods forest stage area, apparently he was able to buy a spirit temple small key twice.

**Comment by TopHatMan** [comment]
I had issues with this but then I came as a child found 2 merchants bought from them fine. But did not see if i could do so twice.

**Comment by r0bd0g** [comment]
Those two scrubs by the forest stage are child-only.

**Comment by friedpen** [comment]
Looking at both this Settings String and the one in Issue #110 I noticed that Shopsanity was set off. I personally haven't experienced despawning issues, even across two 2-player coop runs, but I've only ever done scrubs alongside shopsanity. I don't know if this helps narrow down the issue but I tried two separate generations with minimal settings. No keysanity / songsanity / tokensanity / shopsanity. And another with  no keysanity / songsanity / tokensanity / shopsanity set to 0 items.

OoT Randomizer Version 2.17.5 f.LUM  -  Seed: ScrubsNoShops
Settings (RC8DSATLSU37RBA)

Opening Mido's four chests + kokiri sword chest + buying deku shield I was able to immediately reproduce the despawn issue on LW Sticks scrub (right one before deku mask theatre). 

OoT Randomizer Version 2.17.5 f.LUM  -  Seed: Scrubs0Items
Settings (RC8DSAVLSU37RBA):

Opening Mido's four chests + buying deku nuts in kokiri shop I was unable to reproduce the despawn issue on LW Sticks scrub.
